db/migrate/20240917064038_remove_redundant_global_questions_in_staging.rb

```diff
@@ -0,0 +1,36 @@
+
+# This migration can be removed after it has been deployed to staging
+class RemoveRedundantGlobalQuestionsInStaging < ActiveRecord::Migration[6.1]
+  def up
+    redundant_global_questions.each do |redunant_question, original_question|
+      # Any derived questions are duplicates aswell, so let's destroy them
+      Event::Question.where(derived_from_question_id: redunant_question.id)
+                     .destroy_all!
+
+      redunant_question.destroy!
+    end
+  end
+
+  def down
+  end
+
```

Comment:
This does not seems to be needed, right?

```suggestion
```

Reply:
Wenn man das weglässt, ist die Migration dann aber nicht mehr "rollback"-bar, oder?

Reply:
soweit ich mich erinnere, doch. AFAIR hat die ActiveRecord::Migration vernünftige Default-Implementierungen. Da du direkt `up` verwendet hast, wird glaube ich keine automatische Rollback generiert (wie bei `change`). Kann man aber entweder testen oder einfach drin lassen.